fastlane/jetpack_metadata/it/release_notes.txt

```diff
@@ -0,0 +1,9 @@
+Nella versione 18.0 abbiamo ottimizzati le immagini usate nei Temi, per garantirti download più rapidi e un impatto ridotto sul piano della tua rete dati.
+
+Ora puoi impostare o rimuovere l'immagine in evidenza per un post dalle impostazioni del blocco immagine. E in caso te ne dimenticassi, un nuovo banner in evidenza mostra la selezione corrente.
+
+Anche dal punto di vista del design, i commenti e le notifiche hanno un aspetto più coerente. Nell'ambito della scelta del dominio o del design del sito, le intestazioni sono state ottimizzate per migliorare l'accessibilità sui dispositivi con il testo di grandi dimensioni.
+
+Quando ti confermiamo la pubblicazione di un post tramite un avviso in-app, ora vedrai le e commerciali nel titolo visualizzate correttamente e non in formato HTML.
+
+Last but not least, abbiamo ottimizzato il ripristino da token di accesso invalidati.
```

Comment:
That "Last but not least" untranslated and still in English while the rest is properly in Italian 🤔  Is that intentional, like "the English expression is so commonly known in Italian-speaking countries that it's ok to use it untranslated"? Or more likely a part of the sentence the translator forgot to translate?

Anyway, too late now…

Reply:
I'm also wondering, since this last item is about authentication tokens, was it also relevant to Jetpack in the first place or was that an only-WP change?

I think it might be correct to have kept that for both JP+WP because it's about auth tokens not about the login UI (which indeed differs in JP), but still thought it was worth asking – albeit too late.

Reply:
> I'm also wondering, since this last item is about authentication tokens, was it also relevant to Jetpack in the first place or was that an only-WP change?
>
> I think it might be correct to have kept that for both JP+WP because it's about auth tokens not about the login UI (which indeed differs in JP), but still thought it was worth asking – albeit too late.

@leandroalonso do you know the answer to this? It's too late know but knowing about this will be helpful for me in the future. Thanks!

Reply:
> That "Last but not least" untranslated and still in English while the rest is properly in Italian 🤔 Is that intentional, like "the English expression is so commonly known in Italian-speaking countries that it's ok to use it untranslated"? Or more likely a part of the sentence the translator forgot to translate?

It's an expression that was known to me when I was back in Italy. But that's not saying much, I had an interest for the English language that was above that of my friends, other than those studying English at university, I guess.

Reply:
> @leandroalonso do you know the answer to this? It's too late know but knowing about this will be helpful for me in the future. Thanks!

Yes, in this case the token fix would affect JP+WP. :)